app/models/spaceship.rb

```diff
@@ -111,19 +122,19 @@ def drop_loot
   # Get Storage Capacity of Ship
   def get_storage_capacity
     storage = self.get_attribute('storage')
-    storage = storage + (Spaceship.ship_variables[name]['upgrade']['storage_amplifier']**self.level).round if Spaceship.ship_variables[name]['upgrade']['storage_amplifier'] && (self.level > 0)
+    storage = storage + (self.get_attribute("upgrade.storage_amplifier", default: 1)**self.level).round if (self.level.positive?)
```

Comment:
I think we should keep `> 0` instead of `positive?` here because its easier to read

Reply:
sure... don't have an issue with it  -- replacing `> 0` with `.positive?`  is a fairly recent rubocop recommendation

Reply:
Yeah but Rubocop is not always a 100% where to go from my point of view :) I just think > 0 is much better to read.